README.md

```diff
@@ -0,0 +1,6 @@
+To build and run the tests, you need [stack](https://docs.haskellstack.org/en/stable/README/).
```

Comment:
Perhaps this file was intended to go in the ref/haskell directory rather than the root?